parsl/executors/high_throughput/interchange.py

```diff
@@ -357,6 +358,8 @@ def start(self, poll_period=None):
 
                     try:
                         msg = json.loads(message[1].decode('utf-8'))
+                        if 'reg_time' in msg:
```

Comment:
Other parts of this code assumes that fields always exist and let python's exceptions deal with the case they don't. I think that would be OK to do here too.

Reply:
Fixed.